graphql_api/types/flake_aggregates/flake_aggregates.py

```diff
@@ -1,37 +1,88 @@
-from typing import TypedDict
+from dataclasses import dataclass
 
+import polars as pl
 from ariadne import ObjectType
 from graphql import GraphQLResolveInfo
+from shared.django_apps.core.models import Repository
 
-flake_aggregates_bindable = ObjectType("FlakeAggregates")
+from utils.test_results import get_results
 
 
-class FlakeAggregate(TypedDict):
+@dataclass
+class FlakeAggregates:
     flake_count: int
-    flake_count_percent_change: float | None
     flake_rate: float
-    flake_rate_percent_change: float | None
+    flake_count_percent_change: float | None = None
```

Comment:
dang you didn't like having it grouped huh

Reply:
all args with defaults must come after arguments with no defaults 😭